documentation/tutorial.md

```diff
@@ -1,28 +1,37 @@
-# Charmed MongoDB tutorial:
-Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+# Charmed MongoDB tutorial
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of juju. As a first step this tutorial show you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling TLS. Specifically this tutorial will cover:
+- Setting up your environment using LXD and Juju.
+- Deploying MongoDB using a single command.
+- Accessing the admin database directly.
+- Adding high availability with replication.
+- Changing the admin password.
+- Automatically creating users with relations. 
+- Enabling secure transactions with TLS.
```

Comment:
I would probably change this to be a more active tense. So rather than saying "Specifically this tutorial will cover:" I'd say "In this tutorial we will walk through how to:" (or something more along those lines) and then change to e.g. "Set up your environment...", "Deploy MongoDB using..." etc.

Reply:
(Also, I wasn't sure what "high availability with replication" or "creating users with relations" meant - again, not sure if this is my own lack of familiarity with the product)

Reply:
Great suggestions here! 

The concept of "high availability with replication" is a MongoDB concept that we will expect the user to understand
The concept of "Creating users" should be a MongoDB concept that the user is familiar with. But relations should be something new to them. Does this make things more clear?